src/targets/gpu/include/migraphx/gpu/miopen.hpp

```
@@ -154,6 +155,12 @@ inline tensor_descriptor make_tensor(const migraphx::shape& os, bool pack = fals
         {
             d = miopenInt8;
         }
+#else
+        // MIGraphX on Windows is based on the most recent MIOpen
+        // The support for miopenInt8x4 has been discontinued.
+        (void)pack;
```

We should probably throw an error if pack is set.